### DISCUSSION TITLE
two routed nics, need to connect to security onion dashboard

### DISCUSSION THREAD

**Discussion by ghostrider9899** [Security-Onion-Solutions/securityonion#5480] [Ideas]
Hi,

I currently have two routable nics on my security onion manager.  I currently trying to get to the security onion site with one ip address and it switches to the second ip address. Is there a way to fix this to use the ip address that is able to go to 443 from where i am?

**Comment by dougburks** [comment]
When you ran Setup, it should have asked you how you want to access your installation. For example:
![image](https://user-images.githubusercontent.com/1659467/133068746-1adeafed-8b0d-40ba-9e0c-f5f93c02872d.png)

If you want to change this later, take a look at the following section of our documentation:
https://docs.securityonion.net/en/2.3/url-base.html
